components/nimbus/build.rs

```diff
@@ -8,14 +8,16 @@ pub fn main() {
     #[cfg(feature = "uniffi-bindings")]
     uniffi_build::generate_scaffolding("./src/nimbus.udl").unwrap();
 
+    let python = if cfg!(windows) { "python" } else { "python3" };
```

Comment:
Hmmm will this always be true? Like, if a mac user follows our instructions in the building docs, will they end up with a `python3` binary?

Reply:
Jan-Erik has created a glean-build crate as a wrapper around the glean_parser step, so this script should no longer be needed. I'll be pushing a commit to migrate to this build crate to address your concern here, this was simply a hack to get it to work with Windows development.